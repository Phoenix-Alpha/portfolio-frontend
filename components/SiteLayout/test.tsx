import { shallow } from 'enzyme'
import toJSON from 'enzyme-to-json'

import { SiteLayout } from '.'

describe('<SiteLayout />', () => {
  it('should renders correct <SiteLayout />', () => {
    const children = 'Test children'

    const wrapper = shallow(<SiteLayout>{children}</SiteLayout>)

    expect(wrapper.contains(children)).toBeTruthy()

    expect(wrapper.find('SiteHeader')).toHaveLength(1)

    expect(toJSON(wrapper)).toMatchSnapshot()
  })
})
