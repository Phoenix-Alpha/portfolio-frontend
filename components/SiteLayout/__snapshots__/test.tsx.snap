// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SiteLayout /> should renders correct <SiteLayout /> 1`] = `
<div
  className="flex flex-col min-h-screen"
>
  <SiteHeader />
  Test children
  <SiteFooter
    theme="light"
  />
</div>
`;
